### PR TITLE
add test for AppContentSwitcher component

### LINKED ISSUE #100: (SP: 1) Write unit test for "AppContentSwitcher" component
### Component unit test
Unit test for **"AppContentSwitcher"** component
Scenaries descriptions:
 - [ ] Should render correctly with props
 - [ ] Should call the function "onChange" when it was clicked on the switch
 - [ ] Should render tooltips if the tooltips props are passed

[Link to component](https://github.com/Space2Study-Project/Space2Study-FrontEnd/blob/develop/src/components/app-content-switcher/AppContentSwitcher.jsx)

Current coverage:
Image

### PR DESCRIPTION
**Pull request by LeonidShamarin**
Снимок экрана (2800)1
Снимок экрана (2801)
add test for AppContentSwitcher component

### CODE REVIEW

**Review by AndriiNasadiuk — CHANGES_REQUESTED** [review]
it should be 1 expect function for 1 it block, isn’t it ?

**Comment by LeonidShamarin** [comment]
> it should be 1 expect function for 1 it block, isn’t it ?

Please explain what you mean. I have three blocks, and I need to use more than one expect, this is necessary for exhaustive testing.

**Review by AndriiNasadiuk — APPROVED** [review]

**Review by marianamartsy — APPROVED** [review]
ok

**Review by ArtemHolikov — APPROVED** [review]
Seems good, but why u didn`t placed **switchOptions** function inside test case?

File: tests/unit/components/app-content-switcher/AppContentSwitcher.spec.jsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  const switchOptions = {
+    left: {
+      text: 'left text',
+      tooltip: 'left tooltip'
+    },
+    right: {
+      text: 'right text',
+      tooltip: 'right tooltip'
+    }
+  }
```
Comment (vlad-khrychov) [inline tests/unit/components/app-content-switcher/AppContentSwitcher.spec.jsx]: We can use different props for different test cases for testing all conditions renders. 
Try to provide `switchOptions`  with only left property, only right property, without both and like now

**Review by vlad-khrychov — CHANGES_REQUESTED** [review]

File: tests/unit/components/app-content-switcher/AppContentSwitcher.spec.jsx
Comment (vlad-khrychov) [inline tests/unit/components/app-content-switcher/AppContentSwitcher.spec.jsx:1]: This file is ok

File: tests/unit/containers/tutor-home-page/add-photo-step/AddPhotoStep.spec.jsx
Comment (vlad-khrychov) [inline tests/unit/containers/tutor-home-page/add-photo-step/AddPhotoStep.spec.jsx:1]: Delete this file from PR

File: tests/unit/containers/tutor-home-page/language-step/LanguageStep.spec.jsx
Comment (vlad-khrychov) [inline tests/unit/containers/tutor-home-page/language-step/LanguageStep.spec.jsx:1]: Delete file from PR

**Review by vlad-khrychov — CHANGES_REQUESTED** [review]
After deleting 2 files you can merge this PR without my approval

**Review by vlad-khrychov — APPROVED** [review]

**Review by Mygal-Alex — APPROVED** [review]

**Pull request merged**